eQuiz/modules/eQuiz.Web/Areas/Student/Views/Default/Details.cshtml

```diff
@@ -0,0 +1,65 @@
+﻿
+@{
+    Layout = null;
```

Comment:
I'd suggest to use https://github.com/RDLviv/eQuiz/blob/master/eQuiz/modules/eQuiz.Web/Views/Shared/Layout.cshtml as a layout.